### PR TITLE
Fix launcher_spec tags

### PR DESCRIPTION
**Pull request by pitr-ch**

### CODE REVIEW

File: spec/tags/truffle/launcher_tags.txt [hunk 1]
```diff
@@ -44,23 +44,14 @@ aot:The launcher '-J:cp ' adds the jar
...
-aot:The launcher 'truffleruby' runs when symlinked
-slow:The launcher 'truffleruby' in 'bin' runs when symlinked
+aot:The launcher 'gem' in 'bin' directory runs when symlinked
+aot:The launcher 'irb' in 'bin' directory runs when symlinked
+aot:The launcher 'rake' in 'bin' directory runs when symlinked
+aot:The launcher 'rdoc' in 'bin' directory runs when symlinked
+aot:The launcher 'ri' in 'bin' directory runs when symlinked
+aot:The launcher 'ruby' in 'bin' directory runs when symlinked
+aot:The launcher 'testrb' in 'bin' directory runs when symlinked
+aot:The launcher 'truffleruby' in 'bin' directory runs when symlinked
```
Comment (eregon) [inline spec/tags/truffle/launcher_tags.txt]: We still need the slow tags. They can be added by running `jt test fast spec/truffle/launcher_spec.rb`.

Comment (pitr-ch) [reply spec/tags/truffle/launcher_tags.txt]: they are bellow

File: spec/truffle/launcher_spec.rb [hunk 2]
```diff
@@ -24,7 +24,7 @@
                 truffleruby: /truffleruby .* like ruby 2\.3\.5/ }
 
   launchers.each do |launcher, (test, skip_success)|
-    bin_dirs = [RbConfig::CONFIG['bindir'], *RbConfig::CONFIG['extra_bindirs']]
+    bin_dirs = [File.dirname(RbConfig.ruby), *RbConfig::CONFIG['extra_bindirs']]
```
Comment (eregon) [inline spec/truffle/launcher_spec.rb]: This should be a separate commit, since it's addressing a somewhat different problem.

Comment (pitr-ch) [reply spec/truffle/launcher_spec.rb]: yeah I was in a hurry

File: spec/tags/truffle/launcher_tags.txt (hunk 1 above)
Comment (eregon) [reply spec/tags/truffle/launcher_tags.txt]: This seems to remove
```
slow:The launcher 'truffleruby' in 'bin' runs when symlinked
```
though.

Comment (pitr-ch) [reply spec/tags/truffle/launcher_tags.txt]: It was replaced before by `slow:The launcher 'truffleruby' in 'bin' directory runs when symlinked`
